tests/NuGet.Services.AzureSearch.Tests/SearchDocumentBuilderFacts.cs

```diff
@@ -442,6 +442,36 @@ public async Task SetsExpectedProperties(SearchFilters searchFilters, string exp
 }", json);
             }
 
+            [Theory]
+            [MemberData(nameof(CatalogPackageTypesData))]
+            public async Task SetsExpectedPackageTypes(List<NuGet.Protocol.Catalog.PackageType> packageTypes, string expectedFilterable, string expectedDisplay)
+            {
+                var leaf = Data.Leaf;
+                leaf.PackageTypes = packageTypes;
+
+                var document = _target.UpdateLatestFromCatalog(
+                    SearchFilters.Default,
+                    Data.Versions,
+                    isLatestStable: false,
+                    isLatest: true,
+                    normalizedVersion: Data.NormalizedVersion,
+                    fullVersion: Data.FullVersion,
+                    leaf: leaf,
+                    owners: Data.Owners);
+
+                SetDocumentLastUpdated(document);
+                var json = await SerializationUtilities.SerializeToJsonAsync(document);
+                Assert.Contains(@"
```

Comment:
It might make the code a little more maintainable in these JSON snippet cases to assert the non-serialized values on the document rather than the JSON. I have the single mega-JSON assertion because the shape of the whole document is expected to adhere to a certain contract w.r.t. to a deployed Azure Search index.